cmd/juju/controller/destroy.go

```diff
@@ -232,20 +240,7 @@ func (c *destroyCommand) Run(ctx *cmd.Context) error {
 		return errors.Annotate(err, "getting controller environ")
 	}
 
-	if err := c.ConfirmationCommandBase.Run(ctx); err != nil {
-		return errors.Trace(err)
-	}
-
-	if c.ConfirmationCommandBase.NeedsConfirmation() {
-		updateStatus := newTimedStatusUpdater(ctx, api, controllerEnviron.Config().UUID(), clock.WallClock)
-		modelStatus := updateStatus(0)
-		if err := printDestroyWarning(ctx, modelStatus, controllerName, c.releaseStorage); err != nil {
-			return errors.Trace(err)
-		}
-		if err := jujucmd.UserConfirmName(controllerName, "controller", ctx); err != nil {
-			return errors.Annotate(err, "controller destruction")
-		}
-	}
```

Comment:
I consequence of moving this down is that `Destroying controller` is printed (line 254) before you get confirmation
```
$ juju destroy-controller lxd-head --destroy-all-models
WARNING! This command with destroy the "lxd-head" controller and all it's resources
Destroying controller

 - 1 model will be destroyed
  - model list: "admin/m (alive)"
 - 3 machines will be destroyed
 - 2 applications will be removed
  - application list:
 - 0 filesystem and 0 volume will be destroyed
```